<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Codys-Mac.local" ignored="0" name="Command line test 6e1f81c5-5cdf-43d3-a7f4-8c1978fe2419" tests="2" failures="2" timestamp="27 Mar 2018 07:35:48 GMT" time="727.758" errors="0">
  <testcase name="prankcaller" time="446.678" classname="test.tests.launchApp.Tests">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at test.tests.launchApp.Tests.prankcaller(Tests.java:230)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- prankcaller -->
  <testcase name="launchApp" time="107.359" classname="test.tests.launchApp.Tests">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at test.tests.launchApp.Tests.launchApp(Tests.java:421)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- launchApp -->
</testsuite> <!-- Command line test 6e1f81c5-5cdf-43d3-a7f4-8c1978fe2419 -->
